deploy/prepare_deployment.py

```diff
@@ -151,6 +158,7 @@
     './win_build/Build/x64/Release/slide_show.exe',
     './win_build/Build/x64/Release/test*.exe',
     './win_build/Build/x64/Release/wrappture*.exe',
+    './win_build/Build/x64/Release/collect_data*.exe'
```

Comment:
you can change it to `'./win_build/Build/x64/Release/collect_data.exe'`

Reply:
Also you missed `,` at the end of the line

Reply:
I missed the `,`. Thank you! It solve the issue.